cmd/sync.go

```diff
@@ -134,13 +139,23 @@ func runSync(
 			config.SyncGitignore = &syncFlags.SyncGitignore
 		}
 
+		if setSyncFlags.RemoveOrphaned {
+			config.RemoveOrphaned = &syncFlags.RemoveOrphaned
+		}
+
 		if *config.SyncGitignore {
 			err := exec.UpdateGitignoreIfExists(config)
 			core.CheckIfError(err)
 		}
 
 		err = exec.CloneRepos(config, projects, syncFlags)
 		core.CheckIfError(err)
+
+		// Handle orphaned project removal
+		if *config.RemoveOrphaned || syncFlags.RemoveOrphaned {
```

Comment:
Redudant syncFlags.RemoveOrphaned check here: 

```
if *config.RemoveOrphaned {
	err = exec.RemoveOrphanedProjects(config, projects)
	core.CheckIfError(err)
}
```

should be enough, since the `*config.RemoveOrphaned` has a value always, either defaults to `false` or `true` if set in the mani.yaml.

And if flag is used, the that will set the config file, as u added it above:

```
if setSyncFlags.SyncGitignore {
	config.SyncGitignore = &syncFlags.SyncGitignore
}
```